src/content/docs/fr/develop-and-build.mdx

```diff
@@ -0,0 +1,146 @@
+
+    <Tabs>
+      <TabItem label="npm">
+      ```shell
+      npm run build
+      ```
+      </TabItem>
+      <TabItem label="pnpm">
+      ```shell
+      pnpm build
+      ```
+      </TabItem>
+      <TabItem label="yarn">
+      ```shell
+      yarn run build
+      ```
+      </TabItem>
+    </Tabs>
+
+Astro va construire une version prête à déployer de votre site dans un dossier séparé (`dist/` par défaut) et vous pouvez surveiller son progrès dans la console. Cela vous avertira des erreurs de construction dans votre projet avant de déployer sur le web. Si TypeScript est configuré pour `strict` ou `strictest`, la commande `build` de `build` vérifiera également votre projet pour les erreurs de type.
+
+Lorsque la construction est terminée, exécutez la commande appropriée `preview` (par exemple, `npm run preview`) dans votre terminal et vous pouvez voir la version construite de votre site localement dans la même fenêtre de prévisualisation du navigateur.
+
+Notez que cet aperçu de votre code comme il existait lorsque la commande de construction a été exécutée pour la dernière fois. Cela est conçu pour vous donner une prévisualisation de la façon dont votre site sera affiché lorsqu'il sera déployé sur le web. Toute modification ultérieure apportée à votre code après la construction ne sera **pas** reflétée pendant que vous prévisualisez votre site jusqu'à ce que vous exécutiez à nouveau la commande de construction.
+
+Utilisez (<kbd>Ctrl</kbd> + <kbd>C</kbd>) pour quitter l'aperçu et lancer une autre commande de terminal, comme redémarrer le serveur de développement pour revenir à [travailler en mode développement](#travailler-en-mode-développement) qui se met à jour au fur et à mesure que vous éditez pour afficher un aperçu en direct de vos modifications de code.
+
+<ReadMore>En savoir plus sur le [CLI d'Astro](/fr/reference/cli-reference/) et les commandes sur le terminal que vous utiliserez lorsque vous construirez avec Astro.</ReadMore>
+
+:::tip 
+Vous souhaitez peut-être [déployer votre nouveau site dès maintenant](/fr/guides/deploy/), avant de commencer à ajouter ou de modifier trop de code. Cela est utile pour obtenir une version minimale, fonctionnelle de votre site publié et peut vous économiser du temps et de l'effort de dépannage de votre déploiement ultérieur.
+:::
+
+## Étapes suivantes
+
+Succès ! Vous êtes maintenant prêt à commencer à construire avec Astro ! 🥳
+
+Voici quelques-unes des choses que nous recommandons de vous explorer à l'avenir. Vous pouvez les lire dans n'importe quel ordre. Vous pouvez même laisser notre documentation un peu et jouer dans votre nouvelle base de code de projet Astro, revenant ici à chaque fois que vous rencontrez des difficultés ou que vous avez une question.
```

Comment:
I think this should be `que nous vous recommandons d'explorer` instead. I also suggested a little rephrasing:
```suggestion
Voici quelques éléments que nous vous recommandons d’explorer ensuite. Vous pouvez les lire dans n'importe quel ordre. Vous pouvez même laisser notre documentation un peu et jouer dans votre nouvelle base de code de projet Astro, en revenant ici à chaque fois que vous rencontrez des difficultés ou que vous avez une question.
```